src/Trace/Integrations/Redis.php

```diff
@@ -85,12 +84,11 @@ public static function load($host="")
                 'service.name' => 'redis',
                 'redis.args_length' => count($arguments),
                 'span.kind' => 'client',
+                'db.connection_string' =>  $connection_str,
+                'net.peer.name' => $params->host,
```

Comment:
Can we add test case for this similar to https://github.com/razorpay/opencensus-php/blob/master/tests/unit/Trace/Integrations/PDOTest.php ?

Reply:
added a test